### PR TITLE
docs: add okp4 banner and badges

### PR DESCRIPTION
**Pull request by jeremylgn**
This PR for : 

-  add the OKP4 banner
-  add PRs Welcome badge (see https://makeapullrequest.com/)

 The 'awesome' badge can't be the same height as the other badges...

https://github.com/okp4/awesome/issues/42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
+[![okp4 github banner](./assets/okp4-banner-v2.webp)](https://okp4.network)
+
 # Awesome OKP4
 
 > Awesome OKP4 is a curated list of awesome resources, documents, and tools for [OKP4](https://okp4.network).
 
 [![awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/okp4/awesome)
+ [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://makeapullrequest.com)
```
Comment (ccamel) [inline README.md:8]:
```suggestion
[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=for-the-badge)](https://makeapullrequest.com)
```

**Review by ccamel — CHANGES_REQUESTED** [review]

**Review by ccamel — APPROVED** [review]
:heart:

**Review by antho31 — APPROVED** [review]

**Review by amimart — APPROVED** [review]
👍

**Review by bdeneux — APPROVED** [review]
Nice 👨‍🎨

**Review by ErikssonJoakim — APPROVED** [review]
🔥

**Pull request merged**